pkg/sysregistriesv2/shortnames.go

```diff
@@ -27,12 +28,24 @@ func shortNameAliasesConfPath(ctx *types.SystemContext) (string, error) {
 		return ctx.UserShortNameAliasConfPath, nil
 	}
 
-	configHome, err := homedir.GetConfigHome()
-	if err != nil {
-		return "", err
+	if rootless.GetRootlessEUID() == 0 {
+		// Root user or in a non-conforming user NS
+		return filepath.Join("/var/cache", userShortNamesFile), nil
+	}
+
+	// Rootless user
+	var cacheRoot string
+	if xdgCache := os.Getenv("XDG_CACHE_HOME"); xdgCache != "" {
```

Comment:
Is there a reason to move the non-root configuration as well?

Reply:
Consistency I guess.

Reply:
Yes, that's for consistency.